README.md

```diff
@@ -4,92 +4,108 @@
+
+For projects that already have a keystore file they may already have a certificate as well, but if not you can use the keystore file to generate a certificate:
+```
+keytool -exportcert -keystore driver.keystore -alias driver -file driver_android_certificate.pem
+```
+
+The certificate file should then be placed on a publicly accessible web server at a URL that _uses HTTPS and does not redirect_.
+Storing the certificate file as a publicly readable Amazon S3 object or on GitHub Pages are both good options.
+
+### Build models.jar
+DRIVER uses Gradle to dynamically generate a `models.jar` file that will contain the default schema for the application for the initial model class definition, the backup model classes in case of update failure, and also for running tests.
+
+To trigger Gradle to build this file on a DRIVER environment, determine the UUID of the schema version you want the application to use and in a web browser load the URL  `https://{your-driver-domain}/api/jars/{schema-uuid}/`.
+
+You should get back a 201 Created response while the Gradle process builds the `models.jar` file asynchronously. After a minute or so, refreshing the page should produce a 200 OK response. Once you get that response, if you append `?format=jar` to the URL, it will then download the file.
+
+> **Note** If you do not get a 200 OK response after several minutes, check:
+> - Is `driver-gradle` running on the celery server?
+> - Is there any output in `/var/log/upstart/driver-gradle.log` when you reload the `/api/jars/` URL?
+> If there aren't errors, restarting the service with `sudo service driver-gradle restart` often resolves communication issues. You may need to reload the `/api/jars` URL again after to trigger a new build.
+
+Once downloaded, copy the `models.jar` file to  `app/src/main/assets/models.jar`.
+
+The app requires the `models.jar` file to have been signed by the keystore. You can use `jarsigner` to verify that the models JAR file has been signed correctly:
+```
+$ jarsigner -verify -certs -keystore app/driver.keystore app/src/main/assets/models.jar
+
+jar verified.
+```
+
+### (Optional) Set up Single-Sign-On (SSO)
+This will enable users to sign in with their Google account using OpenID, instead of requiring them
+to have a username and password set up.
+
+> **Note** The Android app requires users to have at least Analyst permissions within DRIVER to use, but users created by SSO default to Public permissions. Users created this way will need to have their account permissions upgraded by a administrator user to be able to use the app.
+
+- Create Android OAuth credentials in the [Google APIs Developer Console](https://console.developers.google.com/)
+- For OAuth to work your environment must already be configured with OAuth credentials for front-end us - it may be appropriate to configure these under the same account.
```

Comment:
I think there's a typo here (`s/us/use`)

Reply:
Good catch, took a look over and I think I found a couple other `a/an` errors that I fixed up as well